backend/curriculum_tracking/tests/test_activity_log_entry_creators.py

```diff
@@ -156,7 +156,31 @@ def test_start_project(self):
 # class log_card_review_request_cancelled_Tests(TestCase):
 
 
-# class log_card_moved_to_complete_Tests(TestCase):
+class log_card_moved_to_complete_Tests(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    def test_card_moved_to_complete_called(self):
+        actor_user = UserFactory(is_superuser=True)
+        card = AgileCardFactory(
+            status=AgileCard.IN_REVIEW,
+            content_item=ProjectContentItemFactory(
+                project_submission_type=ContentItem.LINK, template_repo=None
+            ),
+        )
+        self.login(actor_user)
+
+        add_review_url = f"{self.get_instance_url(card.id)}add_review/"
+        response = self.client.post(
+            add_review_url, data={"status": COMPETENT, "comments": "woohoo"}
+        )
+        self.assertEqual(response.status_code, 200)
+
+        card.refresh_from_db()
+        self.assertEqual(card.status, AgileCard.COMPLETE)
+        self.assertEqual(LogEntry.objects.last().event_type.name, creators.CARD_MOVED_TO_COMPLETE)
+        self.assertEqual(LogEntry.objects.count(), 2)
 
```

Comment:
I might be wrong but maybe to make this test complete and check that all the right things are happening with logging a card moving to complete be good to add assertions that the following are what they should be:

LogEntry.objects.last().object_1 == card.recruit_project
LogEntry.objects.last().object_2 == None
LogEntry.objects.last().actor_user == actor_user
LogEntry.objects.last().effected_user == card.assignee.first()
LogEntry.objects.last().timestamp == card.complete_time

(just double check that I've actually got those right, lol)